### PR TITLE
Add playwright tests

### PR DESCRIPTION
**Pull request by rangelfinal**
Add [Playwright](https://playwright.dev/) based e2e basic tests

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -1,27 +1,3 @@
...
-
-body {
-    color: rgb(var(--foreground-rgb));
-    background: linear-gradient(
-            to bottom,
-            transparent,
-            rgb(var(--background-end-rgb))
-        )
-        rgb(var(--background-start-rgb));
-}
```
Comment (rangelfinal) [inline src/app/globals.css:27]: Default background colors from Next.js boilerplate

File: package.json [hunk 2]
```diff
@@ -5,9 +5,10 @@
     "scripts": {
         "dev": "next dev",
         "build": "next build",
-        "start": "next start",
+        "start": "serve out",
```
Comment (rangelfinal) [inline package.json:8]: `next start` is for dynamic sites
As we use static sites, we can just `serve` the build output

File: e2e/home.spec.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    await page.getByRole("link", { name: "A/B Testing Checklist" }).click();
+    await expect(page).toHaveURL("/checklist/ab-testing-checklist/en");
+
+    // Click on the Header's logo to go back to Home
+    await page.getByRole("link", { name: "Devchecklists logo" }).click();
+    await expect(page).toHaveURL("/");
+
+    // Click on a tag link
+    await page.getByRole("link", { name: "good-practices" }).first().click();
+    await expect(page).toHaveURL("/tag/good-practices");
```
Comment (fjsj) [inline e2e/home.spec.ts:37]: We need a test to check if the tag is really filtering.
We also need: 
- to check if author links are pointing to the right place.
- if non-linked authors really don't have links.
- if vintasoftware link in botton is pointing to the right place.
- if collaborate link is pointing to the right place (right now it's broken)

**Review by fjsj — APPROVED** [review]
LGTM, but I added suggestions for more tests. Please open another PR with them or fix in this one.

**Review by fjsj — APPROVED** [review]

**Pull request merged**